.github/workflows/cd.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    name: Deploy example apps
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: ./.github/actions/setup-cached-python
+
+      - name: Install Modal client package
+        run: pip install modal-client
+
+      - name: Modal token set
+        run: |
+          modal token set --token-id ${{ secrets.MODAL_MODAL_LABS_TOKEN_ID }} --token-secret ${{ secrets.MODAL_MODAL_LABS_TOKEN_SECRET }}
```

Comment:
instead of setting the token, you can just pass the token as an environment variable (`MODAL_TOKEN_ID` and `MODAL_TOKEN_SECRET`)

Reply:
True. I just copied this from our other repo.